examples/hosts/iframe-host/src/inner.ts

```diff
@@ -24,26 +28,36 @@ async function getFluidObjectAndRender(container: Container, div: HTMLDivElement
     view.render(div, { display: "block" });
 }
 
-export async function runInner(divId: string) {
-    const div = document.getElementById(divId) as HTMLDivElement;
+async function loadFluidObject(
+    divId: string,
+    documentId: string,
+    createNew: boolean,
+) {
+    const componentDiv = document.getElementById(divId) as HTMLDivElement;
 
     const documentServiceFactory = await InnerDocumentServiceFactory.create();
-    const module = { fluidExport: TodoContainer };
-    const codeLoader = { load: async () => module };
 
-    const loader = new Loader({
-        urlResolver: documentServiceFactory.urlResolver,
-        documentServiceFactory,
-        codeLoader,
-    });
+    const urlResolver = new InsecureTinyliciousUrlResolver();
 
-    const url = documentServiceFactory.resolvedUrl.url;
-    const documentId = decodeURIComponent(url.replace("fluid://localhost:3000/tinylicious/", ""));
-    const container = await loader.resolve({ url: documentId });
+    const container = await getContainer(
```

Comment:
>getContainer [](start = 28, length = 12)

i don't think this is the right thing to use either, as we should support using an actual code loader here. i'd prefer this use the raw pieces, like loader